main.tf

```diff
@@ -1,7 +1,7 @@
 module "landing_zone" {
   source                  = "./modules/landing_zone"
-  version                 = "0.0.8"
-  root_path               = "${path.module}"
-  landing_zone_providers  = "${var.landing_zone_providers}"
-  landing_zone_components = "${var.landing_zone_components}"
+  root_path               = path.module
+  output_path             = pathexpand("~/.terrahub/output.json")
```

Comment:
please remove `output_path` as input parameter. update the internal scripts to use `~/.terrahub/cache/output/[project_code]/output.json` that can be used across modules ;)